theurgy.lic

```diff
@@ -519,7 +529,7 @@ class TheurgyActions
       complete_or_interrupt_research
       cast_spell({ 'abbrev' => 'bless',
                    'mana' => 1,
-                   'prep_time' => 2,
+                   'prep_time' => 4,
```

Comment:
This is a second place that casts bless, thoughts to move this to its own method?